pkg/volume/util/fsquota/project.go

```diff
@@ -89,7 +89,8 @@ func openAndLockProjectFiles() (*os.File, *os.File, error) {
 	fProjid, err := os.OpenFile(projidFile, os.O_RDONLY|os.O_CREATE, 0644)
 	if err == nil {
 		// Check once more, to ensure nothing got changed out from under us
-		if err := projFilesAreOK(); err == nil {
+		err = projFilesAreOK()
+		if err == nil {
```

Comment:
Thanks for the fix!

Do you think it is better to reorganize the code a little bit to return error immediately instead of return the last line?

Reply:
The error logic was sufficiently nested that I didn't want to pull on any strings that I didn't need to, but I've updated the PR to make for more readable code.